sale_packaging_price/models/sale_stock.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
I rewrite function _product_packaging_change_ that is located into sale_stock module in sale_stock.py file